atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/persistentlock/CheckAndSetExceptionMapper.java

```diff
@@ -32,27 +34,32 @@
 
     @Override
     public Response toResponse(CheckAndSetException ex) {
-        LockEntry actualEntry = extractStoredLockEntry(ex);
-        return createReleaseErrorResponse(actualEntry);
+        String errorId = UUID.randomUUID().toString();
+        log.error("Error handling a request: {}. Stored persistent lock: {}", errorId, extractStoredLockEntry(ex), ex);
+        return createErrorResponse(errorId);
     }
 
     private LockEntry extractStoredLockEntry(CheckAndSetException ex) {
         // Want a slightly different response if the lock was already open
         List<byte[]> actualValues = ex.getActualValues();
         if (actualValues == null || actualValues.size() != 1) {
-            // Rethrow - something odd happened in the db, and here we _do_ want the log message/stack trace.
-            throw ex;
+            // something odd happened in the db
+            return null;
```

Comment:
nit: would it be useful for debugging if we logged the values if actualValues.size() is more than 1? Currently regardless of what's there the exception message will say the current entry is null